.rubocop.yml

```diff
@@ -7,7 +7,7 @@ require:
 AllCops:
   DisplayCopNames: true
   DisplayStyleGuide: true
-  TargetRubyVersion: 2.6
+  TargetRubyVersion: 3.0
```

Comment:
I guess THIS YAML works. Idk though, I'd make it quotes anyway.

Reply:
Yes, RuboCop doesn't consider `3` as "the latest available minor version 3.x", it requires a specific one, and `3 == 3.0`, so it's OK here.

Reply:
Just tested this locally and @AlexWayfer is right. If I change this to 3.3 I get 60+ new offences, so it seems like Rubocop does indeed parse this correctly 😄